capi/geos_c.h.in

```diff
@@ -396,6 +396,14 @@ extern GEOSCoordSequence GEOS_DLL *GEOSCoordSeq_create_r(
     unsigned int size,
     unsigned int dims);
 
+/** \see GEOSCoordSeq_createFromBuffer */
+extern GEOSCoordSequence GEOS_DLL *GEOSCoordSeq_createFromBuffer_r(
+        GEOSContextHandle_t handle,
+        double* buf,
+        unsigned int size,
```

Comment:
shouldn't that be size_t ?

Reply:
Yes, if the other coordinate sequence functions did not use `unsigned int`